css/styles.css

```diff
@@ -131,7 +137,47 @@ nav {
 
 .works {
   display: grid;
-  grid-template: repeat(4, 20rem) / repeat(1, 30rem);
+  grid-template-columns: 1fr;
   min-height: 100vh;
   margin-top: 10rem;
+  justify-items: center;
+}
+
+.card {
+  /* align-items: center; */
```

Comment:
Please, do not leave unused code as comments in your codebase to make your codebase cleaner.